File: Code/Disney/disney.xBandController/src/windows/Test/Disney.xBand.IDMS/Links/Link.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;

namespace Disney.xBand.IDMS.Links
{
    [DataContract]
    public class Link
    {
        [DataMember(Name="href")]
        public String Href { get; set; }

        [DataMember(Name = "name")]
        public String Name { get; set; }
    }
}

File: Code/Disney/disney.xBandController/src/linux/IDMSLib/src/main/java/com/disney/xband/idms/lib/model/Link.java
package com.disney.xband.idms.lib.model;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class Link
{
	
	private String href;
	private String name;
	
	@XmlElement(name="name")
	public String getName()
	{
		return this.name;
	}
	
	@XmlElement(name="name")
	public void setName(String name)
	{
		this.name = name;
	}
	
	

	@XmlElement(name="href")
	public String getHref() {
		return href;
	}

	@XmlElement(name="href")
	public void setHref(String href) {
		this.href = href;
	}

}
